src/bundle/Resources/views/fieldtypes/edit/relation_base.html.twig

```diff
@@ -17,9 +17,9 @@
                 <th class="ez-relations__table-actions">
                     <button
                         {% if limit == 1 %}
-                            data-udw-config="{{ ez_udw_config('single', {}) }}"
+                            data-udw-config="{{ ez_udw_config('object_relation_single', {'starting_location_id' : default_location}) }}"
```

Comment:
Isn't changing the config name a BC break?
//cc @Nattfarinn

Reply:
@dew326 
thanks, fixed :)